src/stylesheets/main.scss

```diff
@@ -46,3 +46,4 @@
 
 /*--- Import Trumps ---*/
 @import 'trumps/helpers';
+@import 'trumps/shame';
```

Comment:
What is this for?

Reply:
Removed: used for testing long form content